agroal-api/src/main/java/io/agroal/api/configuration/supplier/AgroalConnectionFactoryConfigurationSupplier.java

```diff
@@ -98,12 +99,18 @@ public AgroalConnectionFactoryConfigurationSupplier connectionProviderClassName(
         }
     }
 
-    public AgroalConnectionFactoryConfigurationSupplier jdbcTransactionIsolation(TransactionIsolation transactionIsolationLevel) {
+    public AgroalConnectionFactoryConfigurationSupplier jdbcTransactionIsolation(IsolationLevel transactionIsolationLevel) {
```

Comment:
This also breaks binary compatibility; you could add an overload though.

Reply:
On a second though, yes, this method does not need to be relaxed.